components/build/tekton-chains/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+resources:
+# Later we might want to pin this to a version, or download it and split it into
+# its separate pieces, but for now let's just use the official release directly
+- https://storage.googleapis.com/tekton-releases/chains/latest/release.yaml
```

Comment:
I've been burnt with sudden changes in need for privileges - wondering if this a better option ?  https://storage.googleapis.com/tekton-releases/chains/previous/v0.8.0/release.yaml

Reply:
does this have a specific fix we are carrying?

Reply:
I'm happy to set it to 0.8.0 for stability if that's preferred.

Reply:
I am for at least selection of the version in url, so the versioning will be under our control.